qubes_config/global_config.glade

```diff
@@ -451,7 +451,7 @@
                               <object class="GtkImage">
                                 <property name="visible">True</property>
                                 <property name="can-focus">False</property>
-                                <property name="tooltip-text" translatable="yes">This template will be used as default by newly created qubes. It has no influence on existing qubes.</property>
+                                <property name="tooltip-text" translatable="yes">New qubes will be based on this template by default. You can choose a different template when you create a qube. This setting has no effect on existing qubes. &lt;a href="https://www.qubes-os.org/doc/templates/"&gt;Learn more about templates.&lt;/a&gt;</property>
```

Comment:
Not sure if links will work inside of tooltips. If not, let me know.

Reply:
They will not, sadly.

Reply:
No worries. I'll remove the link, then.